Sources/RInAppMessaging/TooltipDispatcher.swift

```diff
@@ -83,6 +87,15 @@ internal class TooltipDispatcher: TooltipDispatcherType, ViewListenerObserver {
             return
```

Comment:
please remove `// TOOLTIP: display permission?` above this line

Reply:
Removed here [067ca5d](https://github.com/rakutentech/ios-inappmessaging/pull/252/commits/458cc054a0abbb56594fdaa1c2cde61ed067ca5d).